sklearn/tests/test_pipeline.py

```diff
@@ -990,27 +991,27 @@ def test_set_feature_union_step_drop(get_names):
     assert_array_equal([[2, 3]], ft.fit_transform(X))
     assert_array_equal(["m2__x2", "m3__x3"], getattr(ft, get_names)())
 
-    with pytest.warns(None) as record:
+    with warnings.catch_warnings(record=True) as record:
```

Comment:
Looking at the commit that added these warnings: https://github.com/thomasjpfan/scikit-learn//commit/16f4ac90f0732988e3b7efe0c937eaff70e99692 , the warnings are not in our codebase anymore. This means we can remove catching the warnings all together.

Reply:
Thanks for the review and the comments!  
I made some new changes.